core/src/main/java/org/apache/gravitino/credential/config/CredentialConfig.java

```diff
@@ -67,6 +67,13 @@ public class CredentialConfig extends Config {
                   false /* reserved */))
           .build();
 
+  public static final ConfigEntry<String> CREDENTIAL_PROVIDERS =
+      new ConfigBuilder(CredentialConstants.CREDENTIAL_PROVIDERS)
+          .doc("Credential providers, separated by comma.")
+          .version(ConfigConstants.VERSION_0_8_0)
+          .stringConf()
+          .create();
```

Comment:
Can you use `toSequence()` for this configuration.

Reply:
updated